### PR TITLE
added traccar GPS tracker to apps/other and backup guide for apps/photo/libre-photos

### PR DESCRIPTION
**Pull request by ippocratis**
Together with the docker-compose I've included usefull XML files but also a way to backup/restore the mysql dB and a way to transfer the location history from Google to traccar.

In librephotos i added a docker-exec to backup and restore the postgres database

Please have a look and tell me what you think about the overall look, if the post is getting too long etc

Its my first pull request so sorry if something is wrong

### CODE REVIEW

**Comment by tborychowski** [comment]
Wow, thanks so much! That traccar is very complicated to setup! I'm sure it will be useful to many people!
Looks good, thanks for contributing!

File: apps/other/traccar.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+## Configuration parameters for MySQL 
+(replace [DATABASE], [USER], [PASSWORD] with appropriate values from docker-compose replace [HOST "IPv4Address" from db_name section in docker network inspect)
+
+```
+<entry key='database.driver'>com.mysql.cj.jdbc.Driver</entry>
+<entry key='database.url'>jdbc:mysql://[HOST]:3306/[DATABASE]?serverTimezone=UTC&amp;allowPublicKeyRetrieval=true&amp;useSSL=false&amp;allowMultiQueries=true&amp;autoReconnect=true&amp;useUnicode=yes&amp;characterEncoding=UTF-8&amp;sessionVariables=sql_mode=''</entry>
+<entry key='database.user'>[USER]</entry>
+<entry key='database.password'>[PASSWORD]</entry>
+```
```
Comment (tborychowski) [inline apps/other/traccar.md:25]: what file is this? where would one put this xml?

File: apps/other/traccar.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+protocol,Time,Latitude,Longitude
+osmand,1,2012-08-25 21:26:20,37.95954620,23.72793730,1
+```
+
+5 columns
+osmand
+1
+time
+lat
+lon
```
Comment (tborychowski) [inline apps/other/traccar.md:179]: what's this text? should it be labelled somehow?

File: apps/other/traccar.md (hunk 1 above)
Comment (ippocratis) [reply apps/other/traccar.md:25]: Traccar uses a configuration file for varius parameters
https://www.traccar.org/configuration-file/
One of this is the dtabase
By default it uses an embeded  h2 db good for testing but not suited for production
An external mysql or postgres db is needed for production
The mysql cofiguretion parameters are here
 https://www.traccar.org/mysql/
Also the db is also a docker container hence the docker network ip is used in the db url

The traccar configuration xml is under /opt/traccar/conf/traccar.xml inside the container and is mounted in host as a volume (in my example /run/media/ippo/TOSHIBA/traccar/conf/traccar.xml)

**Comment by tborychowski** [comment]
Hi,

I merged the PR and wanted to tweak the formatting a bit (sorry, OCD 😉), but I find some places a bit confusing, so I'd appreciate if you could clarify them for me (see comments above).

I also removed:
- the images, as they were linking external images (I'd prefer to keep the repo standalone). 
- paragraph about reverse proxy as this is not required to run the app - I'd like to keep the apps as short and minimal as possible, so that anyone can just (almost) blindly copy & paste and run. 

I hope you don't mind.
(I should probably write-up some contribution guide ;-P)

File: apps/other/traccar.md (hunk 2 above)
Comment (ippocratis) [reply apps/other/traccar.md:179]: It should be more well explained sorry

The csv file generated by the python script contains 3 columns
time latitute longitude
After the sed command it contains 6 (and not 5) columns

@osmand,@deviceid,@Time,@Latitude,@Longitude,@valid

These represent tc_positions table fields in traccar postgres db

osand , 1 , time , lat ,lon AND 1 are the values 
Of the first row in the csv

File: apps/other/traccar.md (hunk 1 above)
Comment (tborychowski) [reply apps/other/traccar.md:25]: ok. understood, thanks!

File: apps/other/traccar.md (hunk 2 above)
Comment (tborychowski) [reply apps/other/traccar.md:179]: great, thanks for clarification. I updated the file. Have a look (if you can) and see if it still makes sense to you :-)

https://github.com/tborychowski/self-hosted-cookbook/blob/master/apps/other/traccar.md

**Pull request merged**